components/ide/jetbrains/image/startup.sh

```diff
@@ -10,6 +10,10 @@ trap "jobs -p | xargs -r kill" SIGINT SIGTERM EXIT
 
 /ide-desktop/status 24000 "$1" &
 
+echo "Desktop IDE: Waiting for the content initializer ..."
+result=$(curl -sS "$SUPERVISOR_ADDR"/_supervisor/v1/status/content/wait/true)
```

Comment:
can it happen that curl times out and we need to retry? we have such logic in the supervisor frontend, i.e. we only continue if a response is json and it has `available: true`: https://github.com/gitpod-io/gitpod/blob/f3ecc7f5cba0d6045e405f5ee15d1ac7fbab0f20/components/supervisor/frontend/src/ide/supervisor-service-client.ts#L53

Reply:
I thought that was overkill since the case of content not being there is so rare anyway. But sure, I can add a loop here.

Reply:
I am just asking, depends on curl, in supervisor frontend we have to do it because connection actually times out. If curl cannot time out then it is overkill. loop would be more robust though